src/routes/home.js

```diff
@@ -37,3 +38,14 @@ module.exports = {
     }
   }
 };
+
+const getUserData = (room, endpoint) => {
```

Comment:
I think this function may need to be declared above the module.exports statement (just in case as getUserData is called before it is defined)